core-web/libs/block-editor/src/lib/extensions/ai-content-prompt/ai-content-prompt.component.ts

```diff
@@ -0,0 +1,98 @@
+import { Component, ElementRef, EventEmitter, Output, ViewChild } from '@angular/core';
+import { FormBuilder, FormGroup, Validators } from '@angular/forms';
+
+import { AiContentService } from '../../shared/services/ai-content/ai-content.service';
+
+@Component({
+    selector: 'dot-ai-content-prompt',
+    templateUrl: './ai-content-prompt.component.html',
+    styleUrls: ['./ai-content-prompt.component.scss']
+})
+export class AIContentPromptComponent {
+    @ViewChild('input') input: ElementRef;
+
+    @Output() formValues = new EventEmitter();
+    @Output() hide = new EventEmitter<boolean>();
+
+    loading = false;
+    form: FormGroup;
```

Comment:
We might need a type for this form. https://angular.io/guide/typed-forms

This type is similar to `formValues` but without the `FormControl` check that doc for more information

Reply:
I miss this totally... :(